README.md

```diff
@@ -1,3 +1,28 @@
+# 1. Introduction
+
+This web application is a [Google Summer of Code 2019](https://summerofcode.withgoogle.com) project under the [MacPorts](https://www.macports.org) organisation.
+
+**Student**: [Arjun Salyan](https://github.com/arjunsalyan) <br>
+**Mentors**: [Mojca Miklavec](https://github.com/mojca), [Umesh Singla](https://github.com/umeshksingla) <br>
+
+### Architecture of the Project
+
+The web application is based on the [Django Framework](http://djangoproject.com) utilising a PostgreSQL database.
+
+The app is supposed to be deployed in a docker container with nginx and uWSGI serving the content.
+
+**Demo**: [AWS EC2](http://ec2-52-34-234-111.us-west-2.compute.amazonaws.com) (running a docker container)
+
+### Goal of the Project
+
+Build an all-in-one application that:
+ 1. displays general information about all the ports
+ 2. fetches and displays build history about the ports
+ 3. collects data from users and displays installation statistics of ports along with the system configuration of users.
```

Comment:
Maybe something like "Collects and displays installation statistics (system configurations and installed ports) from users who opt-it." I would make it clear that user need to explicitly consent rather than having MacPorts doing something on its own.

Reply:
306daae528dc137c9053554c45e90a631ef859490a3ede651d488135602500a3